content/docs/command-reference/root.md

```diff
@@ -10,11 +10,15 @@ usage: dvc root [-h] [-q | -v]
 
 ## Description
 
-This command returns the path to the root directory of the <abbr>DVC
-project</abbr>, relative to the current working directory. It can be used to
-build a path to a dependency, script, or <abbr>data artifact</abbr>, for
-example. Useful when working in a subdirectory of the project, and needing to
-refer to a file in another directory.
+The `dvc root` command returns the path to the root directory of the <abbr>DVC
+project</abbr>, relative to the current working directory.
+
+Using `dvc root`, you can build a path to a dependency, script, or <abbr>data
+artifact</abbr> using `dvc run` from any subdirectory of a <abbr>DVC
+project</abbr>.
+
+You can also use `dvc root` to simplify other commands when working in a
+<abbr>DVC project</abbr>.
```

Comment:
I don't see much meaning in this sentence.

The old one "Useful when working in a subdirectory of the project, and needing to refer to a file in another directory." explained the main use case.

Reply:
Returned this previous version with a minor edit for "needing to".